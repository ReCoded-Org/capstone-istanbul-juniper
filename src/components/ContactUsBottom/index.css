.contactUsBottomContainer {
  background: #2a8c79;
  margin: 0 auto;
}

.contactUsBottomContainer__bottomCurve {
  background: #add4c5;
  border-radius: 50%;
  height: 15vh;
}

.contactUsBottomContainer__bottomCurve__topHalf {
  height: 50%;
  background: #add4c5;
  margin-bottom: 5%;
}

.contactUsBottomContainer__bottomCurve__topHalfImages {
  display: flex;
  flex-wrap: wrap;
  justify-content: space-between;
  align-items: center;
  position: relative;
  top: -10%;
  width: fit-content;
}

.contactUsBottomContainer__bottomCurve__bottomHalf {
  height: 50%;
}

.contactUsBottomContainer__bottomCurve__topHalfImages--mainImg {
  width: 50%;
}

.contactUsBottomContainer__bottomCurve__topHalfImages--subImgOne {
  width: 20%;
}

.contactUsBottomContainer__bottomCurve__topHalfImages--subImgTwo {
  width: 8%;
}

/* Media query rules to override previous rules, as necessary */
/* At the breakpoint only  mainImg and subImgOne are shown */
@media screen and (max-width: 1000px) {
  .contactUsBottomContainer__bottomCurve__topHalfImages {
    top: 0%;
    justify-content: space-around;
  }

  .contactUsBottomContainer__bottomCurve__topHalfImages--mainImg {
    width: 45%;
  }

  .contactUsBottomContainer__bottomCurve__topHalfImages--subImgOne {
    width: 20%;
  }

  .contactUsBottomContainer__bottomCurve__topHalfImages--subImgTwo {
    display: none;
  }
}

/* At the breakpoint only main image is shown */
@media screen and (max-width: 480px) {
  .contactUsBottomContainer__bottomCurve__topHalfImages {
    top: -28%;
  }

  .contactUsBottomContainer__bottomCurve__topHalfImages--mainImg {
    width: 70%;
  }

  .contactUsBottomContainer__bottomCurve__topHalfImages--subImgOne {
    display: none;
  }
}
