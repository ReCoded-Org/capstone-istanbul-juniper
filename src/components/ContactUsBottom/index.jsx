import React from "react";
import contactUsBottomImgRight_0 from "../../images/child.png";
import contactUsBottomImgRight_1 from "../../images/child_2.png";
import contactUsBottomImgLeft from "../../images/family.png";
import "./index.css";

const ContactUsBottom = () => {
  return (
    <div className="contactUsBottomContainer">
      <div className="contactUsBottomContainer__bottomCurve">
        <div className="contactUsBottomContainer__bottomCurve__topHalf">
          <div className="contactUsBottomContainer__bottomCurve__topHalfImages">
            <img
              className="contactUsBottomContainer__bottomCurve__topHalfImages--mainImg"
              alt="Two children playing in the garden"
              src={contactUsBottomImgLeft}
            />

            <img
              className="contactUsBottomContainer__bottomCurve__topHalfImages--subImgOne"
              alt="Girl playing in the garden"
              src={contactUsBottomImgRight_0}
            />

            <img
              className="contactUsBottomContainer__bottomCurve__topHalfImages--subImgTwo"
              alt="Boy playing in the garden"
              src={contactUsBottomImgRight_1}
            />
          </div>
        </div>
        <div className="contactUsBottomContainer__bottomCurve__bottomHalf" />
      </div>
    </div>
  );
};

export default ContactUsBottom;
