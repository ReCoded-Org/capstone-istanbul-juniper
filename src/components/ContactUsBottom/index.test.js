import React from "react";
import ContactUsBottom from "./index.jsx";
import renderer from "react-test-renderer";

it("ContactUsBottom snapshot test", () => {
  const tree = renderer.create(<ContactUsBottom />).toJSON();
  expect(tree).toMatchSnapshot();
});
