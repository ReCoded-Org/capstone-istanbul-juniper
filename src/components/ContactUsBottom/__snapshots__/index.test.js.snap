// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUsBottom snapshot test 1`] = `
<div
  className="contactUsBottomContainer"
>
  <div
    className="contactUsBottomContainer__bottomCurve"
  >
    <div
      className="contactUsBottomContainer__bottomCurve__topHalf"
    >
      <div
        className="contactUsBottomContainer__bottomCurve__topHalfImages"
      >
        <img
          alt="Two children playing in the garden"
          className="contactUsBottomContainer__bottomCurve__topHalfImages--mainImg"
          src="family.png"
        />
        <img
          alt="Girl playing in the garden"
          className="contactUsBottomContainer__bottomCurve__topHalfImages--subImgOne"
          src="child.png"
        />
        <img
          alt="Boy playing in the garden"
          className="contactUsBottomContainer__bottomCurve__topHalfImages--subImgTwo"
          src="child_2.png"
        />
      </div>
    </div>
    <div
      className="contactUsBottomContainer__bottomCurve__bottomHalf"
    />
  </div>
</div>
`;
